### PR TITLE
[Feature] Ads

### PR DESCRIPTION
**Pull request by trmcnvn**
Depends on #380 so we can take advantage of the utility function to check for Pro status.

#### Additional Tasks

- [x] Don't request/render ads for Pro users

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -35,11 +35,6 @@ allprojects {
         configurations.all {
             resolutionStrategy {
                 force 'com.facebook.android:facebook-android-sdk:4.28.0' // Temp fix for broken run-android (see goo.gl/v8u59r)
-                
-                // One signal depends on these, however the new version 12.0 breaks builds
-                force 'com.google.android.gms:play-services-gcm:11.8.0'
-                force 'com.google.android.gms:play-services-analytics:11.8.0'
-                force 'com.google.android.gms:play-services-location:11.8.0'
```
Comment (trmcnvn) [inline android/build.gradle]: @Mikunj Do we need these @ 11.8 for another reason besides broken build? I had no issues with the build today.

Comment (Mikunj) [reply android/build.gradle]: If it doesn't break the build we can go ahead and remove it

**Comment by trmcnvn** [comment]
Wouldn't it make sense for the utility to also handle that case?

**Comment by Mikunj** [comment]
Yep, i'll add it in when i get the chance

**Pull request merged**